package.json

```diff
@@ -1,7 +1,5 @@
 {
-    "name": "containerar",
-    "widgetName": "ContainerAR",
-    "version": "1.0.0",
+    "name": "ar-widgets-native",
```

Comment:
As this security thing raised, I would recommend use `@mendix` scope:
`@mendix/ar-widgets-native`